app.rb

```diff
@@ -0,0 +1,142 @@
+require './person'
+require './student'
+require './teacher'
+require './nameable'
+require './book'
+require './rental'
+require './classroom'
+
+class App
+  attr_reader :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
```

Comment:
OPTIONAL
- [ ] You can fix the typo in `persons`. It should be `people`

Reply:
Thanks a lot, You are right at this point :100: